lib/models/device.coffee

```diff
@@ -1268,6 +1268,36 @@ getDeviceModel = (deps, opts) ->
 		.asCallback(callback)
 
 	###*
+	# @summary Generate a device key
+	# @name generateDeviceKey
+	# @public
+	# @function
+	# @memberof resin.models.device
+	#
+	# @param {String|Number} uuidOrId - device uuid (string) or id (number)
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.device.generateDeviceKey('7cf02a6');
+	#
+	# @example
+	# resin.models.device.generateDeviceKey(123);
+	#
+	# @example
+	# resin.models.device.generateDeviceKey('7cf02a6', function(error) {
+	# 	if (error) throw error;
+	# });
```

Comment:
Done - changed to match examples that have a result (I think I copied from a set that didn't)